src/hooks/useManageTokens.tsx

```diff
@@ -76,7 +76,13 @@ const TokenList: React.FC<TokenListProps> = ({ tokens, onToggleToken, disabledTo
               onToggleToken(address, !checked)
             }}
           >
-            <OptionItem name={name} symbol={symbol} image={image}>
+            <OptionItem
+              name={name}
+              symbol={symbol}
+              image={image}
+              opaque={overrideReason === 'DEPRECATED'}
```

Comment:
Would it be good to move the deprecated flag to the token details so you don't need to repeat this logic?

Reply:
My thinking was, there could be overrides for reasons other than deprecation